scripts/build-merge.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+ PART_ADDR=$(./gen_esp32part.py images/partition-table.bin | grep vfs | awk -F',' '{print $4}')
+ esptool.py --chip esp32 merge_bin -o images/merged-firmware.bin --flash_mode dio --flash_size 16MB --flash_freq 40m 0x1000 images/bootloader.bin 0x8000 images/partition-table.bin 0x10000 images/micropython.bin 0x510000 images/ota_data_initial.bin "${PART_ADDR}" images/sys.img
```

Comment:
For the partitions that are movable.
ota_0,app,ota_0,0x10000,2560K,
otadata,data,ota,0x510000,8K,
I would prefer to get them from the partition table like the VFS. Also we can set some variables for the ones that are
pre-defined.
BOOT_ADDR=0x1000
PART_ADDR=0x8000
OTA_ADDR=$(.gen_esp32part.py images/partition-table.bin | grep ota_0 | awk -F',' '{print $4}')
OTA_DATA_ADDR=$(.gen_esp32part.py images/partition-table.bin | grep otadata | awk -F',' '{print $4}')
Then the esptool.py command can look like this.
esptool.py --chip esp32 merge_bin -o images/merged-firmware.bin --flash_mode dio --flash_size 16MB --flash_freq 40m "${BOOT_ADDR}" images/bootloader.bin "${PART_ADDR}" images/partition-table.bin "${OTA_ADDR}" images/micropython.bin "${OTA_DATA_ADDR}" images/ota_data_initial.bin "${PART_ADDR}" images/sys.img

Reply:
I agree movable partitions ...  I was going to address that with the next round of changes to handle multiple builds

Reply:
I am OK with that.